a10_neutron_lbaas/v2/objmodel.txt

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Did you want to leave this file in there?  No harm, just curious if it was something you wanted to keep in the repo.